packages/compass/src/main/logging.ts

```diff
@@ -120,6 +120,11 @@ async function setupLogging(compassApp: typeof CompassApplication) {
       return app.getPath('logs');
     });
 
+    // TODO: there must be a better place to put this
```

Comment:
No idea where would be the best place to put this.

Reply:
yeah, it doesn't really belong here, what's wrong about putting it in its own file?

Reply:
which package?